pkg/util/log/logutils.go

```diff
@@ -49,11 +50,11 @@ func SetLog(logLevel string, logfile bool, logFileName string) {
 	})
 }
 
-type MyFormatter struct {
+type myFormatter struct {
```

Comment:
Please avoid types and variables with their name starting `My...`.
It is in the open so `My` seems inappropriate to me.
If there is a specific thing the Formatter does (e.g., colored output, ...), consider using that.

Reply:
Done